README.md

```diff
@@ -1,2 +1,33 @@
 # flask-container-scaffold
 Common base layer for Flask applications that are deployed in containers.
+
+
+### Setting up a development environment
+
+You may set up your environment with virtualenv or another preferred tool for
+managing virtual environments, but here are some directions for doing so using
+[pipenv](https://pipenv.pypa.io/en/latest/). First, install pipenv:
+  pip install --user pipenv
+
+Next, using it to set p your development environment:
+  pipenv update -d
+
+If you prefer to use pip directly in your venv, simply specify the
+requirements.txt and test-requirements.txt files.  There is also a
+dist-requirements.txt, if you will be building the project for distribution.
+
+Any remaing directions will assume you are either in your venv, which for
```

Comment:
"remaining"

I'm not sure that the "either" belongs here, since the alternative is introduced in a separate sentence.